src/monika-config-schema.json

```diff
@@ -83,6 +83,51 @@
         }
       }
     },
+    "mongo": {
+      "title": "MongoDB",
+      "type": "array",
+      "description": "Monitor MongoDB health ",
+      "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "required": [],
+        "properties": {
+          "uri": {
+            "title": "MongoDB URI",
+            "description": "The hostname or IP address of the MongoDB server",
+            "type": "string",
+            "examples": ["mongodb://127.0.0.1:27017"],
+            "default": ""
+          },
+          "host": {
+            "title": "MongoDB host",
+            "description": "The hostname or IP address of the MongoDB server",
+            "type": "string",
+            "examples": ["localhost", "127.0.0.1"],
+            "default": ""
+          },
+          "port": {
+            "title": "Port number",
+            "description": "Port number used for the MongoDB",
+            "type": "integer",
+            "examples": ["27017"],
+            "default": ""
```

Comment:
would it be good to have default to 27017?

Reply:
Good idea. Updated